linux-tkg-config/prepare

```diff
@@ -348,7 +348,7 @@ _set_cpu_scheduler() {
   if [ -z "$_cpusched" ]; then
     msg2 "Which CPU sched variant do you want to build/install?"
     msg2 "Project C (pds) / BMQ (bmq) is usually a good balance for gaming."
-    msg2 "Select \"cfs\" (linux kernel's default) if unsure."
+    msg2 "Select \"eevdf\" (linux kernel's default) if unsure."
```

Comment:
I think we should define a `_default_scheduler` variable and populate it when the kernel version is chosen (`cfs` or `eevdf`). then just use it everywhere where it makes sense, like here.